debug/org.eclipse.debug.tests/src/org/eclipse/debug/tests/launching/LaunchManagerTests.java

```diff
@@ -83,8 +82,8 @@ public void testGenerateValidName() {
 	public void testGenerateConfigNameReservedName() {
 		if(Platform.OS_WIN32.equals(Platform.getOS())) {
 			String configname = "aux"; //$NON-NLS-1$
-			String name = getLaunchManager().generateUniqueLaunchConfigurationNameFrom(configname);
-			assertEquals("config name should be 'aux'", configname, name); //$NON-NLS-1$
+			String generatedConfigurationName = getLaunchManager().generateUniqueLaunchConfigurationNameFrom(configname);
+			assertEquals("config name should be 'aux'", configname, generatedConfigurationName); //$NON-NLS-1$
```

Comment:
```suggestion
			assertEquals("config name should be '" + configname + "'", configname, generatedConfigurationName); //$NON-NLS-1$ //$NON-NLS-2$
```

Reply:
I complete removed the unnecessary duplication of the expected names from the error messages in 9635d79.